### ISSUE TITLE
Check whether testing and validation contains the real rating information

### ISSUE THREAD

**Issue by mengzaiqiao** [beta-team/beta-recsys#81]
### Describe the bug
Please provide a clear and concise description of what the bug is. If applicable, add screenshots to help explain your problem, especially for visualization related problems.

### To Reproduce
Please provide a [Minimal, Complete, and Verifiable example](https://stackoverflow.com/help/mcve) here. We hope we can simply copy/paste/run it. It is also nice to share a hosted runnable script (e.g. Google Colab), especially for hardware-related problems.

### Describe your attempts
- [ ] I checked the documentation and found no answer
- [ ] I checked to make sure that this is not a duplicate issue

You should also provide code snippets you tried as a workaround, StackOverflow solution that you have walked through, or your best guess of the cause that you can't locate (e.g. cosmic radiation).

### Context
 - **OS** [e.g. Windows 10, macOS 10.14]: 
 - **Hardware** [e.g. CPU only, GTX 1080 Ti]: 
 - **Environment** [e.g. Pytorch 1.4, Tensorflow 2.0]:

In addition, figure out your used Beta-Recsys version. Then attach the corresponding label on the issue.

### Additional Information
Other things you want the developers to know.
